src/ai-authorization/infrastructure/refresh-token-user-meta-repository.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Yoast\WP\SEO\AI_Authorization\Infrastructure;
+
+use RuntimeException;
+use Yoast\WP\SEO\Helpers\User_Helper;
+
+/**
+ * Class Refresh_Token_Repository
+ * Handles the storage and retrieval of refresh tokens for users.
+ */
+class Refresh_Token_User_Meta_Repository implements Refresh_Token_User_Meta_Repository_Interface {
```

Comment:
Nit: I think this could benefit from some documentation in the class that its about a `refresh token` and not to refresh the token. I was very confused when I only saw a save and remove in a class that starts with `Refresh`